ui/admin/app/components/form/field/list-wrapper/key-value/index.hbs

```diff
@@ -41,15 +40,15 @@
           </B.Td>
 
           <B.Td>
-            {{#if @removeOptionByIndex}}
+            {{#if this.removeOptionByIndex}}
               <Hds::Button
                 data-test-remove-button
                 @text={{t 'actions.remove'}}
                 @color='critical'
                 @icon='trash'
                 @isIconOnly={{true}}
                 disabled={{@disabled}}
-                {{on 'click' (fn @removeOptionByIndex index)}}
+                {{on 'click' (fn this.removeOptionByIndex @name index)}}
```

Comment:
Why pass in the name here? You already have access to it in the function

Reply:
yup not needed!